ci/github-script/bot.js

```diff
@@ -213,16 +230,43 @@ module.exports = async ({ github, context, core, dry }) => {
     // Check for any human reviews other than the PR author, GitHub actions and other GitHub apps.
     // Accounts could be deleted as well, so don't count them.
     const reviews = (
-      await github.paginate(github.rest.pulls.listReviews, {
-        ...context.repo,
-        pull_number,
-      })
-    ).filter(
+      await github.graphql.paginate(
+        `query($owner: String!, $repo: String!, $pr: Int!, $endCursor: String) {
+        repository(owner: $owner, name: $repo) {
+          pullRequest(number: $pr) {
+            reviews(first: 100) {
+              nodes {
+                state
+                user: author {
+                  ... on User {
+                    login
+                    id: databaseId
```

Comment:
I don't quite understand this bit. My understanding was `id` in graphql-land is the node id while REST-land id is the database id. This looks like it is mapping databaseId to nodeId, which smells off. But I'm probably misunderstanding something.